app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id] if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
phân biệt present? và nil? khác nhau thế nào nhé

Reply:
Dạ chị. Cả present? và nil? đều được dùng cho object nhưng
nil? là check nil của object trả về true nếu object nil
Còn present? là check not nill, not false, not empty của object và trả về true nếu object not nill, not false, not empty

Reply:
dùng present? thay cho kết hợp !.nil? nha em